### DISCUSSION TITLE
레벨1 글쓰기 "한 문장 쓰기" | 2022.03.16

### DISCUSSION THREAD

**Discussion by Wonmi91** [woowacourse/retrospective#9] [General]
### 레벨 1 글쓰기 주제 : 우아한테크코스 한 달 생활기

* 레벨 1 글쓰기의 첫 문장을 지금 생각나는대로 써보자!
* 최종 글쓰기에 꼭 이 문장을 반영할 필요는 없다 : )

#### 이전 기수 크루들은 첫 문장을 어떻게 썼을까?

* ‘우아한테크코스에서의 한 달 생활기’를 주제로 글을 쓰라는 미션을 받고는 무엇을 써야 할지, 누구를 대상으로 글을 써야 할지 정하기 어려웠다. 
* "나? 요즘 안 바빠… 그냥 조금 정신이 없네."
* 우테코에는 내 이름이 없다.
* “그래서 뭘 배웠어?” 얼마 전 우아한 테크코스에 대해 지인에게 이야기하던 도중 받았던 질문이다. 
* 당신은 잘 지내시나요? 코로나 때문에 시작된 재택 교육이 계속 길어지네요.

**Comment by jayjaehunchoi** [comment]
네오의 레벨 1 강의를 처음 맛본 26살 먹은 후니는 X을 지렸다.

**Reply by kbsat** [reply]
우승

**Reply by jinyoungchoi95** [reply]
우승

**Reply by syoun602** [reply]
인정

**Reply by woowahan-neo** [reply]
팩트체크 가능한가요?

**Reply by Ohzzi** [reply]
이게뭐죠?

**Reply by onschan** [reply]
비키지않고 뭐해! 얼른 올려드려!

**Reply by bugoverdose** [reply]
후니! 후니! 후니! 후니! 후니! 후니! 후니!

**Comment by soyi47** [comment]
잠이 확 줄었다.

**Comment by skullkim** [comment]
죽을거 같은 행복함.(힘들어 죽을거 같은데 행복하다)

**Reply by her0807** [reply]
스컬은 진짜 진짜임

**Comment by Byeongju-Kong** [comment]
모든 것이 새롭게 느껴지고, 나 자신도 새로워진다.

**Comment by intae92** [comment]
김 빠진 콜라.

**Reply by her0807** [reply]
우주먼지맛 코카콜라 후속작 우테코맛 코카콜라

**Comment by onschan** [comment]
여름이었다.

**Comment by nan-noo** [comment]
우테코에 붙을 줄 몰랐다.

**Comment by yeon-06** [comment]
"너 배민 취직했어?"

**Comment by moonheekim0118** [comment]
닮고 싶은 사람들을 많이 만났다.

**Comment by DWL21** [comment]
온라인에서 오프라인으로 오프라인에서 온라인으로 간다.

**Comment by yhh1056** [comment]
지금은 새벽이다.

**Comment by YJGwon** [comment]
인생은 체리픽이 안되니까.

**Reply by ldk980130** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 명언 인정

**Comment by juunzzi** [comment]
전 팀에서 나온 후 개발이 하기 싫어졌어 그 마음을 개선해보려고 들어왔는데 진짜로 내 마음이 나아지고있더라

**Comment by ulimy** [comment]
매일 기나긴 하루를 보냈는데, 왜 한 달이나 지났지?

**Comment by kth990303** [comment]
바쁘지만 재밌는 시간을 보내는 새내기로 돌아간 느낌이다.

**Comment by sinb57** [comment]
나는 무엇이 변했을까

**Comment by Ohzzi** [comment]
"이제 배민 시키면 너가 오냐?"

**Reply by progress0407** [reply]
ㅋㅋㅋㅋㅋㅋ 아 이거 저도 들었습니다

**Comment by ldk980130** [comment]
우테코를 시작하기 전과 나는 제법 다르다.

**Comment by jinyoungchoi95** [comment]
우테코에서 오프라인같은 온라인 생활이 시작됐다.

**Comment by progress0407** [comment]
"그래서 거긴 재밌어?? 할만 해?" 전 회사 분들의 궁금증이 담긴 연락이 종종 온다.

**Comment by nailseong** [comment]
집에 있지만 집에 가고 싶다.

**Reply by her0807** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by yunjin-kim** [comment]
옆에서 강아지가 코를 골고 있다

**Comment by rladpwl0512** [comment]
좀 못할수도 있지. 내가 짱이야!

**Comment by gudonghee2000** [comment]
오늘도 자다가 블랙잭 코드를 수정하는 악몽을 꿧다. 눈뜨니까 데일리 미팅 10분전이다.

**Comment by kamwoo** [comment]
서울은 많이 춥네...

**Comment by hwangstar156** [comment]
"바쁘다.. 겨울이였다.."

**Comment by yangdongjue5510** [comment]
돌이켜보면 믿기지 않는 한달이었다.

**Comment by woong7** [comment]
"배민 상품권 주냐?". 주변에서 가장 많이 들었던 말이었다.

**Reply by soyi47** [reply]
222...

**Comment by jurlring** [comment]
리뷰어님이 이틀 째 리뷰를 안 남겨주신다.

**Reply by pocojang** [reply]
구구가 갑니다

**Reply by jurlring** [reply]
앗 그건...

**Reply by YJGwon** [reply]
업계포상 on

**Reply by RunaNam** [reply]
주디는 좋겠네요

**Comment by cjlee38** [comment]
나는 무엇을 바라보고 있었을까?

**Comment by frontend-jihyeok-um** [comment]
"너 배달하냐? 근데 그걸 평생직업하긴 좀 그렇지않아?"

**Reply by her0807** [reply]
저도 많이 들었는데 해리도 들었군요

**Comment by hyeonic** [comment]
한달이 이렇게 빨리 지나가다니

**Reply by her0807** [reply]
순삭 순살 매트

**Comment by jaejae-yoo** [comment]
중심을 잘 잡고 있을까?

**Comment by HJ-Rich** [comment]
우테코가 어떻냐고요? 저의 경험 중 최고였어요.

**Reply by bugoverdose** [reply]
이거지

**Reply by her0807** [reply]
이 한문장으로 다음 문장들이 필요 없게 되었네요.

**Comment by Seongwon97** [comment]
대학 4년보다 더 값진 한 달이었다.

**Comment by al-bur** [comment]
잘하고 있는지 잘 모르겠다

**Comment by jin7969** [comment]
빨리 출입증 가지고 싶다!!

**Comment by kbsat** [comment]
가야 할 때가 언제인가를 분명히 알고 가는 이의 뒷모습은 얼마나 아름다운가.

**Reply by woowahan-neo** [reply]
제게도 꿈은 있었습니다

**Comment by usageness** [comment]
"이게 맞나?"

**Comment by uk960214** [comment]
우테코의 진정한 매력은 이곳에서 만나는 사람들이다.

**Comment by BETTERFUTURE4** [comment]
혼자 동굴 속에서 코딩하던, 과거의 내가 기억나지 않는다.

**Comment by jswith** [comment]
짜릿해! 늘 새로워

**Comment by verus-j** [comment]
너무 너무 너무 재밌다.

**Reply by pocojang** [reply]
너무 x 3 => 준

**Comment by JangBomi** [comment]
한 달이 굉장히 빠르게 지나갔다.

**Comment by Seungpang** [comment]
10개월의 마라톤을 진행하기 위한 확실한 준비운동 시작!

**Comment by sojukang** [comment]
우아 우아 하게 만들어줘

**Reply by HJ-Rich** [reply]
최고최고 ㅋㅋㅋㅋ

**Reply by sanaandmomo** [reply]
말만하지 말고~

**Comment by byhhh2** [comment]
안녕 우테코~ 나야

**Reply by pocojang** [reply]
누구세요?

**Reply by HJ-Rich** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by 2yujeong** [comment]
정리의 필요성을 느꼈다.

**Comment by shindong96** [comment]
궁금한 사람들이 많네요.

**Comment by LAH1203** [comment]
"우리 언제 만나?"

**Comment by SuyeonChoi** [comment]
모두 우테코에 있었군!

**Comment by jojogreen91** [comment]
내가 우테코에? 이왜진?

**Comment by RunaNam** [comment]
흰바탕에 검은글씨가 조금은 어색한 날이 왔다.

**Comment by lokba** [comment]
헬스보다 코딩이 1순위가 된 지금...

**Reply by hwangstar156** [reply]
💪🏿

**Reply by pocojang** [reply]
💪

**Comment by prefer2** [comment]
혼자보다는 여럿이 더 좋아.

**Reply by pocojang** [reply]
같이 힘들자~

**Comment by kyukong** [comment]
하루하루를 보내다보니 한 달이 지났다.

**Comment by compy-ryu** [comment]
![so](https://user-images.githubusercontent.com/94832610/158505698-5bdc66d3-ec06-41f9-99de-53543e9600ba.png)

**Reply by HJ-Rich** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Reply by YJGwon** [reply]
정성추

**Reply by usageness** [reply]
추천.. 추천이오..

**Comment by bcc0830** [comment]
어라..어째서 나... 눈물이..?

**Comment by cndqjacndqja** [comment]
오프라인.. 올해 안에 할 수는 있을까?

**Comment by Hongdonggeon** [comment]
왜 한줄 쓰는 것도 고민하고 있지

**Comment by sure-why-not** [comment]
오 좋은데로 가시나봐요?

**Reply by HJ-Rich** [reply]
앗 공감...

**Comment by KangYunHo1221** [comment]
벌써 머리가 많이 자랐다.

**Comment by injoon2019** [comment]
한 달 전의 나에게.

**Comment by syoun602** [comment]
하루하루가 즐겁다.

**Comment by RIANAEH** [comment]
얼른 브이로그에 쓸 소스를 찍어야하는데 점점 찍을게 없다.

**Comment by leo0842** [comment]
까만 우테코 보며 꿈을 키웠지 언젠가는 나도 저길 가게 될 거야

**Comment by yh20studio** [comment]
부쩍 금요일 저녁이 좋아졌다.

**Comment by jhy979** [comment]
우테코가 뭐야? 어~ 배민! 아~

**Comment by devHudi** [comment]
학습의 방향이 잡혀가는 요즘이다.

**Comment by dayelop** [comment]
지우학? 2521?

**Comment by soominsohn** [comment]
우테코는 재밌다.

**Comment by dongho108** [comment]
함께 자라기

**Reply by her0807** [reply]
(데일리와) 함께 자라기

**Comment by HyeonbinSa** [comment]
같이 하니 재밌다.

**Reply by HJ-Rich** [reply]
와 첫문장이 이러면 너무 읽고 싶네요 ㅎㅎ

**Comment by kimchan123** [comment]
이게 공부지~!

**Comment by wishoon** [comment]
서울 생활은 어때?

**Comment by Juhyung990122** [comment]
살면서 처음으로 개발자가 될 수 있을지 의심됐다.

**Comment by JUDONGHYEOK** [comment]
벌써 한달

**Comment by woose28** [comment]
아니? 벌써 한 달이 지났다고?!

**Comment by sanaandmomo** [comment]
![8](https://user-images.githubusercontent.com/38878617/158505751-efa7ed31-0f99-4be6-a510-2a843d7494cb.jpg)
생각보다 힘들어서 암흑 진화했다.

**Reply by pocojang** [reply]
응애

**Comment by hyewoncc** [comment]
비몽사몽 흐린 눈으로 게더를 키면 또 하루가 시작된다.

**Comment by her0807** [comment]
**i'm on the next level yeah**

**Reply by HJ-Rich** [reply]
와... 이거다... 👍

**Reply by YJGwon** [reply]
"ㄷ"

**Reply by Ohzzi** [reply]
절대적 룰을 지켜

**Reply by her0807** [reply]
@Ohzzi  절대적 컨벤션 지켜

**Comment by euijinkk** [comment]
열정의 지표는 대개 온도보다 지속성이다.

**Comment by tonic523** [comment]
한 줄 쓰는 것조차 어려운 우테코

**Comment by chawani** [comment]
내가 어떤 사람인지 가장 많이 생각해 본 한 달이었다.

**Comment by hanull** [comment]
생각보다 힘들다. 생각보다 재밌다.

**Comment by awesomeo184** [comment]
한 달이 지났다.

**Comment by yeongunheo** [comment]
내 20대의 마지막을 우테코와 함께하기로 했다.

**Comment by OzRagwort** [comment]
아 행복하다... ! 😃

**Comment by liswktjs** [comment]
한달동안 협동하는 법, 에러 코드 생성하는 것, 에러 코드 수정하는 것을 배우는 한달이였다.

**Comment by wonsss** [comment]
<왜요>가 없으면 아무것도 못하던 삶에서, 하고 싶으면 하는 삶으로

**Comment by hamcheeseburger** [comment]
눈 감았다 뜨니 3월

**Comment by airman5573** [comment]
문 손잡이에 먼지가 쌓였다.

**Comment by greenblues1190** [comment]
한달간의 try catch

**Reply by pocojang** [reply]
`finally` 도 껴줘요

**Reply by Ohzzi** [reply]
try with resources를 사용해주세요

**Comment by EastHShin** [comment]
조금 힘들어도 괜찮아, 혼자가 아니기에

**Comment by hyunrrr** [comment]
빨리 오프라인 했으면 좋겟다!

**Comment by yaho99** [comment]
한달째 작고 소중한 내 쓰레기를 리팩터링하는 중이다.

**Comment by nbalance97** [comment]
뉴비로 환생한 기분이었다.

**Reply by kth990303** [reply]
저두요

**Comment by kwannee** [comment]
나는 생각보다 게으르지 않았구나

**Reply by hwangstar156** [reply]
게으른데용

**Reply by bugoverdose** [reply]
부럽네요

**Comment by brorae** [comment]
하루하루 꾸준해지는 느낌이다.

**Comment by summerlunaa** [comment]
최고로 알찼던 한 달간의 방콕생활

**Comment by kkojae91** [comment]
지금의 난 어디에..? 나도 나를 믿지 못한다.

**Comment by Gyuchool** [comment]
그때 그 시절

**Comment by DomMorello** [comment]
내 인생에 또 다시 재밌는 일이 생겼다.

**Comment by yxxnghwan** [comment]
버닝 이벤트.

**Comment by is2js** [comment]
붙여주면 고등학교 때처럼 할 수 있어? 어. 나 우테코 붙으면 돌(아온)범(생)이 될 수도 있어

**Comment by bugoverdose** [comment]
하아... 글 써야 하는데...

**Comment by sc0116** [comment]
배고프다

**Comment by seong-wooo** [comment]
자고싶어요

**Comment by InKyoJeong** [comment]
하루가 정말 빨리 지나간다.

**Comment by Yboyu0u** [comment]
우테코 생각보다 재밌네요. ㅎ

**Comment by Hyunta** [comment]
지금 나는 잘하고 있을까?

**Comment by meatsby** [comment]
![62D22B36-9F17-4861-9235-7A10F08A31DA](https://user-images.githubusercontent.com/83967672/158506182-5e9ded76-a14f-49e7-80cd-b97451bc5051.png)
리팩터링하면 그만이야..

**Reply by pocojang** [reply]
무야호~

**Comment by tco0427** [comment]
"우테코? 그게 뭐야? 회사다니는거야?"

**Comment by Youngyoon-1** [comment]
새로운 시작 우테코

**Reply by her0807** [reply]
칙테코 화이팅

**Comment by asebn1** [comment]
힘든데 재밌다

**Comment by ronci** [comment]
난 어떻게 우테코에 들어왔지?
